global.ini

```diff
@@ -61485,24 +61485,24 @@ mobiglas_CryAstro_TransactionFailed=Ihre Transaktion ist fehlgeschlagen. Bitte v
 mobiglas_CryAstro_Unavailable=An Ihrem Standort sind keine Dienste verfügbar
 mobiglas_CryAstro_WelcomeLocation=Willkommen bei %s. Bitte Wähle eine Dienstleistung aus:
 mobiglas_OutlawThanksPlayer_Sub_Title=Ruto
-mobiglas_OutlawThanksPlayer_Text=Was habe ich dir gesagt? Einfach richtig? Hacken Sie ein Terminal und schwupps, senken Sie die Kriminalitätsrate. Scheint, als wären wir ein ziemlich gutes Team. Ich denke, du tust. \n\nWer weiß, vielleicht bekommen wir irgendwann wieder die Chance, zusammenzuarbeiten. \n\n-Ruto
+mobiglas_OutlawThanksPlayer_Text=Was habe ich dir gesagt? Einfach richtig? Hacken Sie ein Terminal und schwupps, senken Sie den CrimeStat. Scheint, als wären wir ein ziemlich gutes Team. Ich denke, du tust. \n\nWer weiß, vielleicht bekommen wir irgendwann wieder die Chance, zusammenzuarbeiten. \n\n-Ruto
 mobiglas_OutlawThanksPlayer_Title=Hat wunderbar funktioniert
 mobiglas_Yela_Advisory_Body=***ACHTUNG***\n\nfür alle Schiffe in der Nähe von Yela (Stanton 2c) gilt eine Reisewarnung. \n\nObwohl das Gebiet stark von Crusader Security überwacht wird, sind derzeit bekannt, dass Gesetzlose in dem Gebiet operieren und Angriffe immer noch möglich sind. Die meisten Hinterhalte wurden im Asteroidenfeld in der Nähe von Yela gemeldet. Crusader Security empfiehlt Reisenden dringend, die Risiken für ihre persönliche Sicherheit sorgfältig abzuwägen und Reisen in die Region nach Möglichkeit ganz zu vermeiden. \n\nDieser Reisehinweis wurde von Sasha Rust, Sicherheitsdirektorin von Crusader Security, autorisiert.
 mobiglas_Yela_Advisory_Title=Yela Reisehinweis
 mobiglas_ui_Appointment=Termin
-mobiglas_ui_BountyHunter=Kopfgeldjäger
+mobiglas_ui_BountyHunter=Kopfgeld \ Bountys
 mobiglas_ui_ECN=ECN-Alarm
```

Comment:
/ anstatt \
Backslash ist meist ein Sonderzeichen in Übersetzungen. Slash ist korrekt.